modules/core/mwaa/stack.py

```diff
@@ -192,6 +192,20 @@ def __init__(
                         f"arn:aws:dynamodb:{self.region}:{self.account}:table/addf-{deployment_name}-{module_name}*"
                     ],
                 ),
+                aws_iam.PolicyStatement(
+                    actions=["sagemaker:*"],
+                    effect=aws_iam.Effect.ALLOW,
+                    resources=[
+                        f"arn:aws:sagemaker:{self.region}:{self.account}:*"
+                    ],
+                ),
+                aws_iam.PolicyStatement(
+                    actions=["iam:PassRole"],
```

Comment:
Also PassRole on "*" should be addressed

Reply:
I restricted this to only passing roles to sagemaker service